### ISSUE TITLE
[couchbase] greetings repository always returns empty list

### ISSUE THREAD

**Issue by ignasi35** [lagom/lagom-recipes#63]
In both [couchbase recipes](https://github.com/lagom/lagom-recipes/tree/master/couchbase-persistence) the `/api/user-greetings` endpoint always returns `[]`.

/cc @johanandren

**Comment by ignasi35** [comment]
Just noticed the warning:

```
23:33:26.910 [warn] couchbase-journal.read(akka://hello-impl-application) [sourceThread=hello-impl-application-lagom.persistence.dispatcher-30, akkaTimestamp=22:33:26.910UTC, akkaSource=couchbase-journal.read(akka://hello-impl-application), sourceActorSystem=hello-impl-application] - Missing the [tags-ordering] index, the events by tag query will not work without it, se plugin documentation for details
```

**Comment by ignasi35** [comment]
To test the Couchbase recipes locally I followed the instructions on `README.md`. 


I cloned https://github.com/akka/akka-persistence-couchbase and used 

```
$ docker-compose -f docker/couchbase-1-node/docker-compose.yml up --build
```

as specified in the [docs](https://doc.akka.io/docs/akka-persistence-couchbase/current/developing.html#running-dockerized-couchbase-for-tests).

The issue is that code runs using the `1.0-RC1` version while I've started Couchbase using a `HEAD` version of `docker/couchbase-1-node/docker-compose.yml` which changed since `1.0-RC1` where `tags-ordering` is no longer present.


Recipes should Just Work ™️.

**Comment by johanandren** [comment]
Yeah, cloning master is not a good idea, no guarantee that will match what it is built on.
